cmd/kubeadm/app/cmd/join.go

```diff
@@ -57,6 +57,16 @@ func NewCmdJoin(out io.Writer) *cobra.Command {
 		"(required) Shared secret used to secure bootstrap. Must match the output of 'kubeadm init'",
 	)
 
+	cmd.PersistentFlags().Int32Var(
```

Comment:
Should this be part of the non-flag arg that is passed in?  That would feel more natural.

```
kubeadm join --token=XXX 1.2.3.4:1234?
```